### PR TITLE
KEP for Deployment Finished Event Extension with Deployment Endpoint URL

### PR DESCRIPTION
**Pull request by grabnerandi**
See my proposal for this KEP 0005!

### CODE REVIEW

File: text/0005-deployment-finished-with-deployed-endpointurl.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+No breaking changes as we just add a new optional field.
+
+## Prior art and alternatives
+
+Not aware of any alternatives other than our current "guestimage" approach
+
+## Open questions
+
+How shall we name that field?
+Is one field enough or shall we provide a set of fields, e.g: root endpoint, health check endpoint ...?
```
Comment (christian-kreuzberger-dtx) [inline text/0005-deployment-finished-with-deployed-endpointurl.md]: I think one field with the base url is enough, anything else is subject to configuration of the testing service (e.g., jmeter-service).

Comment (johannes-b) [reply text/0005-deployment-finished-with-deployed-endpointurl.md]: I agree with using only one field and like the already proposed property name: `deployedEndpoint`.

**Comment by johannes-b** [comment]
I like the proposed KEP, as it helps to separate concerns. (i.e., it is not the test-service's responsibility to come up with the endpoint to test).

File: text/0005-deployment-finished-with-deployed-endpointurl.md (hunk 1 above)
Comment (christian-kreuzberger-dtx) [reply text/0005-deployment-finished-with-deployed-endpointurl.md]: How about `deploymentURL`, or to be even more generic (it does not need to be a URL): `deploymentURI`?

Comment (grabnerandi) [reply text/0005-deployment-finished-with-deployed-endpointurl.md]: deploymentURI works for me!

Comment (johannes-b) [reply text/0005-deployment-finished-with-deployed-endpointurl.md]: The team agreed on introducing two properties: `deploymentURILocal` and `deploymentURIPublic` (to allow tests on dark deployments). Please read the update in the section *Internal details*.

**Pull request merged**